content/en/tracing/trace_collection/open_standards/otel_collector_datadog_exporter.md

```diff
@@ -10,319 +10,338 @@ further_reading:
 
-Run an Opentelemetry Collector container to receive traces either from the [installed host](#receive-traces-from-host), or from [other containers](#receive-traces-from-other-containers).
+## Using Docker
+
+Run an Opentelemetry Collector container to receive traces either from [localhost](#receive-traces-from-host), or from [other containers](#receive-traces-from-other-containers).
 
 <div class="alert alert-info">
 The latest tag of the OpenTelemetry Collector Contrib distro <a href="https://github.com/open-telemetry/opentelemetry-collector-releases/issues/73">is not updated on every release</a>.
 Pin the Collector to the latest version to pick up the latest changes.
 </div>
 
-#### Receive traces from host
+### Receive traces from localhost
 
-1. Create a `otel_collector_config.yaml` file. [Here is an example template](#ingesting-opentelemetry-traces-with-the-collector) to get started. It enables the collector's OTLP receiver and the Datadog exporter.
+Follow these steps if you wish to run the OpenTelemetry Collector as a Docker image and receive traces from the same host:
 
-2. Choose a published Docker image such as [`otel/opentelemetry-collector-contrib:latest`][10].
+1. Create a `collector.yaml` file. You may use the [example template above](#configuring-the-datadog-exporter) to get started.
 
-3.  Determine which ports to open on your container. OpenTelemetry traces are sent to the OpenTelemetry Collector over TCP or UDP on several ports, which must be exposed on the container. By default, traces are sent over OTLP/gRPC on port `4317`, but common protocols and their ports include:
+2. Choose a published Docker image such as [`otel/opentelemetry-collector-contrib:<VERSION>`][10].
 
-      - Zipkin/HTTP on port `9411`
-      - Jaeger/gRPC on port `14250`
-      - Jaeger/HTTP on port `14268`
-      - Jaeger/Compact on port (UDP) `6831`
-      - OTLP/gRPC on port `4317`
-      - OTLP/HTTP on port `4318`
+3. Determine which ports to open on your container. OpenTelemetry traces are sent to the OpenTelemetry Collector over HTTP on two ports, depending on whether you are using gRPC or not. These ports must be exposed on the container. By default, traces are sent over gRPC on port 4317.
 
-4. Run the container with the configured ports and an `otel_collector_config.yaml` file. For example:
+4. Run the container and expose the necessary port, using the previously defined `collector.yaml` file. For example, considering you are using port 4317:
 
       ```
       $ docker run \
-      -p 4317:4317 \
-      --hostname $(hostname) \
-      -v $(pwd)/otel_collector_config.yaml:/etc/otelcol-contrib/config.yaml \
-      otel/opentelemetry-collector-contrib:<VERSION>
+          -p 4317:4317 \
+          --hostname $(hostname) \
+          -v $(pwd)/otel_collector_config.yaml:/etc/otelcol-contrib/config.yaml \
+          otel/opentelemetry-collector-contrib:<VERSION>
       ```
```

Comment:
indent this 3 spaces (should align with the R in Run in the first sentence).

Reply:
Well, actually this is supposed to be a tab, so 4 spaces. It does not have to align with `run`.